schemas/session-replay/common/pointer/pointer-event-type-schema.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "session-replay/common/pointer/pointer-event-type-schema.json",
+  "title": "PointerEventType",
+  "description": "Schema of a PointerEventType",
+  "readOnly": true,
+  "oneOf": [
+    {
+      "$ref": "pointer-down-schema.json"
```

Comment:
Instead of using dedicated types for these I would rather use enums and even ints with a description for each int property. This will save us some space and also will be more optimised when having to serialize the events in the mobile sdk. We are already experiencing some issues with this in Android SDK and would like whenever is possible to simplify the models a bit. @0xnm

Reply:
Ok after a discussion with Benoît, it seems that the enum is a good approach:
I will use string values for the [pointer-event-type-schema.json](https://github.com/DataDog/rum-events-format/pull/116/files/a482e7ac419971d17d6889a98517b9b5d875ce64#diff-fe73ab245b110f6ec5345524eae5ed135e929051a24970a06ac7633cf12c1c6a) as it is more intuitive as integers.

Note that the downside of this approach is that we will have to introduce a constant `PointerEventType` as

`export const PointerEventType: {
  PointerDown: number,
  PointerUp: number,
  PointerMove: number
} = {
  PointerDown: 0,
  PointerUp: 1,
  PointerMove: 2
} as const`

That will not consume the generated type as the following example:

`export const RecordType: {
  BrowserFullSnapshot: typeof BrowserRecordType.FullSnapshot
  BrowserIncrementalSnapshot: typeof BrowserRecordType.IncrementalSnapshot
  ...
} = {
  BrowserFullSnapshot: 2,
  BrowserIncrementalSnapshot: 3,
  ...
} as const`